test_e2e.sh

```diff
@@ -124,25 +141,25 @@ GIT_SYNC \
     --repo="$REPO" \
     --root="$ROOT" \
     --dest="link" > "$DIR"/log."$TESTCASE" 2>&1 &
-sleep 2
+sleep 3
 assert_link_exists "$ROOT"/link
 assert_file_exists "$ROOT"/link/file
 assert_file_eq "$ROOT"/link/file "$TESTCASE 1"
 # Move forward
 echo "$TESTCASE 2" > "$REPO"/file
 git -C "$REPO" commit -qam "$TESTCASE 2"
-sleep 2
+sleep 3
```

Comment:
I tried to find a time that didn't flake but also wasn't an eternity.  Is this flaking for you?

Reply:
Not anymore, using the three second time-out it is stable